### PR TITLE
Feat/stream audio

### PR DESCRIPTION
**Pull request by Sabin-Subedi**

### CODE REVIEW

File: src/lib/elevenlabs/index.tsx
Comment (nabing8) [inline src/lib/elevenlabs/index.tsx:1]: Is this used anymore, lets remove the file if not.

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "db:studio": "drizzle-kit studio"
   },
   "dependencies": {
+    "@11labs/react": "^0.0.4",
```
Comment (nabing8) [inline package.json:19]: Where is this used?

**Review by nabing8 — DISMISSED** [review]

File: src/components/cat-widget.tsx [hunk 2]
```diff
@@ -91,7 +92,8 @@ export default function CatChatWidget({ propertyFeatures, className }: CatChatWi
                       animate={{ opacity: 1, y: 0 }}
                       transition={{ duration: 0.3 }}
                     >
-                      <Typography variant="pUiMedium">{text}</Typography>
+                      <StreamTextAudio text={text} />
+                      {/* <Typography variant="pUiMedium">{text}</Typography> */}
```
Comment (lucerowb) [inline src/components/cat-widget.tsx:96]: cat reply ma pani same garnu parla, but dky reply aairako xaina

File: src/lib/elevenlabs/index.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+"use client";
```
Comment (lucerowb) [inline src/lib/elevenlabs/index.tsx:1]: might not need this file

File: package.json (hunk 1 above)
Comment (lucerowb) [inline package.json:19]: also this package is not used

**Review by lucerowb — APPROVED** [review]
LGTM

File: src/components/cat-widget.tsx [hunk 4]
```diff
@@ -105,10 +125,32 @@ export default function CatChatWidget({ propertyFeatures, className }: CatChatWi
...
+                                    delay: index * 0.2,
+                                    duration: 0.5,
+                                  },
+                                }),
+                              }}
+                            >
+                              <Button
+                                className="w-full"
+                                variant="outline"
+                                onClick={() => handleAnswer({ text, action })}
```
Comment (lucerowb) [inline src/components/cat-widget.tsx:149]: Need to pass on whole option for the cat to reply with value rather than only text and action

**Review by lucerowb — CHANGES_REQUESTED** [review]
Implementation looks fine but one comment to be resolved

File: src/components/cat-widget.tsx (hunk 4 above)
Comment (Sabin-Subedi) [reply src/components/cat-widget.tsx:149]: i think @nabing8 is working on that part

Comment (lucerowb) [reply src/components/cat-widget.tsx:149]: This was done in a commit directly before this, might be rebased and mixed up

**Review by lucerowb — APPROVED** [review]

**Pull request merged**